src/components/Navbar.js

```diff
@@ -64,35 +67,23 @@ const ButtonNotification = ({ isLoggedIn }) => {
 };
 
 /** View component for navbar
- * @param {object} props
- * @param {boolean} props.isLoggedIn
- * @param {any} props.logoutUserAction
  */
-const NavbarView = ({ isLoggedIn = false, logoutUserAction }) => (
-  <AppBar>
-    <Toolbar className='nav-container'>
-      <CustomButton tip='Home' component={Link} to='/'>
-        <Icon.Home />
-      </CustomButton>
-      <ButtonNotification isLoggedIn={isLoggedIn} />
-      <ButtonLogInOut isLoggedIn={isLoggedIn} logout={logoutUserAction} />
-      <ButtonRegister isLoggedIn={isLoggedIn} />
-    </Toolbar>
-  </AppBar>
-);
-
-const mapStateToProps = ({ user }) => {
-  const isLoggedIn = !!user && !!user.userInfo && !!user.userInfo.handle;
-  return {
-    isLoggedIn,
-  };
-};
-
-const mapActionsToProps = {
-  logoutUserAction,
+export const Navbar = () => {
+  const { isAuthenticated } = useUserAuthenticationData();
+  const { logoutUser } = useUserLogout();
+  return (
+    <AppBar>
+      <Toolbar className='nav-container'>
+        <CustomButton tip='Home' component={Link} to='/'>
+          <Icon.Home />
+        </CustomButton>
+        <ButtonNotification isAuthenticated={isAuthenticated} />
+        <ButtonLogInOut
+          isAuthenticated={isAuthenticated}
+          logoutUser={logoutUser}
+        />
+        <ButtonRegister isAuthenticated={isAuthenticated} />
+      </Toolbar>
+    </AppBar>
+  );
```

Comment:
There's yeeting... and then there's death star laser.

Reply:
BOOM